From 4e22c9eb10b04295d3b02c4ac609b8b481dda3e8 Mon Sep 17 00:00:00 2001
From: Baptiste Robert <baptistex.robert@intel.com>
Date: Thu, 8 Aug 2013 17:18:27 +0200
Subject: Mixer: remove unused variables

BZ: 129540
[CLEAN]

Currently, some variables are unused in mixer.c.

This patch removes these variables.

Category: aosp improvement
Domain: AudioComms-General
Origin: internal
Upstream-Candidate: yes
Change-Id: Ib689e590c6ab21939d8228f0f0fa7bd44357e53e
Orig-Change-Id: Idaa1325db7a935464e2c020126e31885351e7a1a
Signed-off-by: Baptiste Robert <baptistex.robert@intel.com>
---
 mixer.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/mixer.c b/mixer.c
index f074ed5..dd62c9c 100644
--- a/mixer.c
+++ b/mixer.c
@@ -91,10 +91,9 @@ void mixer_close(struct mixer *mixer)
 struct mixer *mixer_open(unsigned int card)
 {
     struct snd_ctl_elem_list elist;
-    struct snd_ctl_elem_info tmp;
     struct snd_ctl_elem_id *eid = NULL;
     struct mixer *mixer = NULL;
-    unsigned int n, m;
+    unsigned int n;
     int fd;
     char fn[256];
 
-- 
1.7.4.1

